exploration/test_repo_manager.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from remote_gym.repo_manager import RepoManager
+
+WORKING_DIR = Path(".repo_cache")
+REPOSITORY = "ssh://git@stash.wargaming.net:2222/rp43/synthetic-player-environments.git"
+
+
+def test_default():
+    RepoManager(WORKING_DIR).get(REPOSITORY)
+
+
+def test_update_default():
```

Comment:
How is this test different from `test_update_default()`?
Or is the test here, that you execute it twice?

If yes, then this test does not work *on its own* and you should add another same line.